src/sql/workbench/services/notebook/browser/sql/sqlSessionManager.ts

```diff
@@ -159,7 +159,7 @@ export class SqlSession implements nb.ISession {
 }
 
 class SqlKernel extends Disposable implements nb.IKernel {
-	private _queryRunner: QueryRunner;
+	private _queryRunners: Map<string, QueryRunner> = new Map<string, QueryRunner>();
```

Comment:
Shouldn't we be canceling/disposing the query runners when the kernel is disposed?